### PR TITLE
Add support for more media message types: Video, Document and Audio

### PR DESCRIPTION
**Pull request by Shamkh**
# Description
Add support for more media message types
Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/communication/azure-communication-messages/CHANGELOG.md [hunk 1]
```diff
@@ -22,5 +12,14 @@ This is the GA release of Azure Communication Messages Python SDK. For more info
 - Using `MessageTemplateClient`
   - Get list of all WhatsApp templates.
 
+## 1.0.1 (2024-10-10)
```
Comment (arifibrahim4) [inline sdk/communication/azure-communication-messages/CHANGELOG.md:15]: Generally, keep latest version changes log entry at the top. I see same for SDK module.

**Review by arifibrahim4 — APPROVED** [review]

File: sdk/communication/azure-communication-messages/CHANGELOG.md (hunk 1 above)
Comment (Shamkh) [reply sdk/communication/azure-communication-messages/CHANGELOG.md:15]: For Python , This need to be at the bottom. Build is failing if I am moving it anywhere else.

Comment (glorialimicrosoft) [reply sdk/communication/azure-communication-messages/CHANGELOG.md:15]: Correct, this should at the top, and the other langauges are using 1.1.0 as the version numbe rinstead of 1.0.1, is that something Python specifically adheres to? If not, could we align with the same version number?

Comment (arifibrahim4) [reply sdk/communication/azure-communication-messages/CHANGELOG.md:15]: I see other SDK module having latest version entry at the top. Below is some example

https://github.com/Shamkh/azure-sdk-for-python/blob/7e904912629037ca8733cda82dc1e7cde4a091c6/sdk/communication/azure-communication-sms/CHANGELOG.md?plain=1#L3

https://github.com/Shamkh/azure-sdk-for-python/blob/7e904912629037ca8733cda82dc1e7cde4a091c6/sdk/communication/azure-communication-phonenumbers/CHANGELOG.md?plain=1#L3

I am not sure why our module alone will have build failure when other module has same structure.

Comment (Shamkh) [reply sdk/communication/azure-communication-messages/CHANGELOG.md]: version is updated to 1.1.0

**Review by maximrytych-ms — APPROVED** [review]

**Pull request merged**